.buildkite/packaging.pipeline.yml

```diff
@@ -172,6 +174,73 @@ steps:
         artifact_paths:
           - build/distributions/**/*
 
+      - label: "SNAPSHOT: {{matrix}} Linux/amd64 FIPS"
+        env:
+          PLATFORMS: "${PLATFORMS_FIPS}"
```

Comment:
```suggestion
          PLATFORMS: "${PLATFORMS_AMD64_FIPS}"
```

Reply:
Bad rename from Goland, I'll fix it right away!

Reply:
Fixed in e1cf0bd091e854c0a8a936fe6dfb234c0cea98ad